docker-compose.override.sample.yaml

```diff
@@ -7,4 +7,9 @@ services:
       context: .
       target: tester
     volumes:
-      - ./extension:/src/extension
+      - ./runtime/docker-entrypoint.d:/docker-entrypoint.d
+      - /home/amorvan/dev/nextcloud/server/apps:/var/www/html/apps:ro
+      - /home/amorvan/dev/nextcloud/server/lib:/var/www/html/lib:ro
```

Comment:
Maybe remove your name and put a relative path for the nexcloud server  place ?

Reply:
Done with some sort of standard relative path.